### ISSUE TITLE
Display formatting not correct when used on PowerShell

### ISSUE THREAD

**Issue by Liampor** [tldr-pages/tldr-python-client#178]
Hello,

### Problem
When I use TLDR on PowerShell, the output formatting is not correct, it looks like
![image](https://user-images.githubusercontent.com/19495916/142961754-b30b2be1-f424-4ad7-8f00-6354960338e7.png)

###  Environment
PowerShell version 7.2.0
TLDR is installed via `conda install  tldr`
```
$ tldr -v
>>> tldr 2.0.0 (Client Specification 1.4)

$ pip install tldr
>>> Requirement already satisfied: tldr in d:\tsusersoftware\miniconda3\lib\site-packages (2.0.0)
>>> Requirement already satisfied: argcomplete in d:\tsusersoftware\miniconda3\lib\site-packages (from tldr) (1.12.3)
>>> Requirement already satisfied: termcolor in d:\tsusersoftware\miniconda3\lib\site-packages (from tldr) (1.1.0)
>>> Requirement already satisfied: colorama in d:\tsusersoftware\miniconda3\lib\site-packages (from tldr) (0.4.4)
```

### Others
TLDR works well on git-bash with the same installation and system

![image](https://user-images.githubusercontent.com/19495916/142962222-ed0020cd-bffc-4fd0-ac8d-e11d828941c7.png)

How could I fix it? Thank you all

**Comment by MasterOdin** [comment]
Are you able to change the following line:

https://github.com/tldr-pages/tldr-python-client/blob/d8722698284e5adde7fc561ea1c0b6ab114df528/tldr.py#L490

to be:

```python
    colorama.init(strip=options.color, convert=True)
```

and see if that fixes the problem?

**Comment by MasterOdin** [comment]
Alternatively, we may need to adjust it so that we do a `from sys import stdout` after calling `colorama.init()` to get it to work on Windows. I'm not totally sure, hopefully the `convert` thing above works as that's easier to manage.

**Comment by Liampor** [comment]
Thanks, but I tried to modify the init-function and delete the local cache,  but same issue again. 😂

> Are you able to change the following line:
> 
> https://github.com/tldr-pages/tldr-python-client/blob/d8722698284e5adde7fc561ea1c0b6ab114df528/tldr.py#L490
> 
> to be:
> 
> ```python
>     colorama.init(strip=options.color, convert=True)
> ```
> 
> and see if that fixes the problem?

**Comment by Liampor** [comment]
Update:
I have installed Fluent Terminal now and still use PowerShell 7.0 as Shell, then the tldr could work as expected.

![image](https://user-images.githubusercontent.com/19495916/143172140-d9d331d6-8147-4e90-a185-13521c5f9dc4.png)

**Comment by ImportTaste** [comment]
Interestingly, colors work when piping to Write-Output even with the default console.

**Comment by ImportTaste** [comment]
Until this is fixed, I came up with a little workaround that'll work on all consoles.

```powershell
filter tldr {[String]::Join([Environment]::NewLine,( tldr.exe $Args ))}
```

You can also pipe to Write-Output for the same effect.

**Comment by kbdharun** [comment]
Hi, since the past few releases, the display formatting (for colors) has been fixed in PowerShell [so it works on both older versions and in PowerShell 7] (not sure why the issue wasn't closed yet)

![Image](https://github.com/user-attachments/assets/40d8c4d7-1e6a-443f-8a5f-6d84d528dbcd)

![Image](https://github.com/user-attachments/assets/e17bf9d7-ab49-46d4-b734-7a3e634d49e8)

> Question though, does tldr --help print out formatted text on Linux, like the npm version does? Or is this just something that was done for the Python client in general?

All our official clients supports `tldr --help` option to show command usage regardless of the platform.
